src/ugeneui/transl/russian.ts

```diff
@@ -1556,38 +1556,42 @@ Now you can type ugene in command line to start UGENE.</translation>
 <context>
     <name>U2::ProjectImpl</name>
     <message>
-        <location filename="../src/project_support/ProjectImpl.cpp" line="105"/>
+        <location filename="../src/project_support/ProjectImpl.cpp" line="103"/>
         <source>Project URL is set to %1
 </source>
         <translation>Адрес проекта изменён : %1</translation>
     </message>
     <message>
-        <location filename="../src/project_support/ProjectImpl.cpp" line="115"/>
+        <location filename="../src/project_support/ProjectImpl.cpp" line="113"/>
         <source>Project contains NULL document</source>
         <translation>Project contains NULL document</translation>
     </message>
     <message>
-        <location filename="../src/project_support/ProjectImpl.cpp" line="125"/>
+        <location filename="../src/project_support/ProjectImpl.cpp" line="123"/>
         <source>Adding document to the project: %1</source>
         <translation>Adding document to the project: %1</translation>
     </message>
     <message>
-        <location filename="../src/project_support/ProjectImpl.cpp" line="147"/>
+        <location filename="../src/project_support/ProjectImpl.cpp" line="145"/>
         <source>Find document failed during resource locking</source>
         <translation>Find document failed during resource locking</translation>
     </message>
     <message>
-        <location filename="../src/project_support/ProjectImpl.cpp" line="153"/>
+        <location filename="../src/project_support/ProjectImpl.cpp" line="151"/>
+        <source>Not enough resources for load document, resource: &apos;%1&apos; available: %2%3 requested: %4%3</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
         <source>Not enough resources for load document, resource name: &apos;%1&apos; available: %2%3 requested: %4%3</source>
-        <translation>Not enough resources for load document, resource name: &apos;%1&apos; available: %2%3 requested: %4%3</translation>
+        <translation type="vanished">Not enough resources for load document, resource name: &apos;%1&apos; available: %2%3 requested: %4%3</translation>
```

Comment:
Why do we have 'vanished' in the fixed version. If the translation is vanished it should be removed, right?